app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::OrganizationsController < Api::ApiController
+  include FilterByOwner
+  include FilterByCurrentUserRoles
+  # include TranslatableResource
+  include IndexSearch
+  include AdminAllowed
+
+  require_authentication :update, :create, :destroy, scopes: [:organization]
+
+  resource_actions :show, :index, :create, :update, :deactivate
+  schema_type :json_schema
+
+  prepend_before_action :require_login,
+    only: [:create, :update, :destroy]
+
+  def create
+    organizations = Organization.transaction do
+      Array.wrap(params[:organizations]).map do |organization_params|
+        Organizations::Create.with(api_user: api_user).run!(organization_params)
+      end
+    end
+
+    created_resource_response(organizations)
+  end
+
+  def update
+    Organization.transaction do
+      Array.wrap(resource_ids).zip(Array.wrap(params[:organizations])).map do |organization_id, organization_params|
```

Comment:
@marten, what do you think? It was explained to me that the schema says the API should support an array of params, but it isn't totally clear to me from the routes. Is this to maintain the standard, is this an upgrade, or is the wrap n' zip here unnecessary?